src/Platform/PlatformKey.cpp

```diff
@@ -0,0 +1,112 @@
+#include <cassert>
+#include <unordered_map>
+
+#include "Utility/String.h"
+#include "Utility/MapAccess.h"
+
+#include "PlatformKey.h"
+
+static std::unordered_map<PlatformKey, std::string> displayNameByKey = {
+    {PlatformKey::F1,           "F1" },
```

Comment:
this is a bit debatable in my opinion
it's really a game logic level representation that's used in a single game menu (key bindings/controls?)
i dont think it belongs to the platform

Reply:
True, it doesn't, missed this one.

I've added a TODO, will move it out in a separate PR.